src/Sylius/Component/Translation/Factory/TranslatableFactoryInterface.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Component\Translation\Factory;
+
+use Sylius\Component\Resource\Factory\FactoryInterface;
+
+/**
+ * Interface for factories creating translatable resources.
+ *
+ * @author Paweł Jędrzejewski <pawel@sylius.org>
+ */
+interface TranslatableFactoryInterface extends FactoryInterface
```

Comment:
Okay, seems right. I'm not much into all the translation stuff, but will there be any problem, if someone calls `createForLocale` two times for the same base entity but different languages?